nutanix/config.go

```diff
@@ -13,27 +14,32 @@ const Version = "3.1"
 
 // Config ...
 type Config struct {
-	Endpoint    string
-	Username    string
-	Password    string
-	Port        string
-	Insecure    bool
-	SessionAuth bool
-	WaitTimeout int64
-	ProxyURL    string
+	Endpoint           string
+	Username           string
+	Password           string
+	Port               string
+	Insecure           bool
+	SessionAuth        bool
+	WaitTimeout        int64
+	ProxyURL           string
+	FoundationEndpoint string
+	FoundationPort     string
 }
 
 // Client ...
 func (c *Config) Client() (*Client, error) {
 	configCreds := client.Credentials{
-		URL:         fmt.Sprintf("%s:%s", c.Endpoint, c.Port),
-		Endpoint:    c.Endpoint,
-		Username:    c.Username,
-		Password:    c.Password,
-		Port:        c.Port,
-		Insecure:    c.Insecure,
-		SessionAuth: c.SessionAuth,
-		ProxyURL:    c.ProxyURL,
+		URL:                fmt.Sprintf("%s:%s", c.Endpoint, c.Port),
+		Endpoint:           c.Endpoint,
+		Username:           c.Username,
+		Password:           c.Password,
+		Port:               c.Port,
+		Insecure:           c.Insecure,
+		SessionAuth:        c.SessionAuth,
+		ProxyURL:           c.ProxyURL,
+		FoundationEndpoint: c.FoundationEndpoint,
+		FoundationPort:     c.FoundationPort,
+		FoundationURL:      fmt.Sprintf("%s:%s", c.FoundationEndpoint, c.FoundationPort),
```

Comment:
Do we need FoundationURL as separate member of client.Credentials ?
Whenever we need the foundation URL we can always create it using endpoint and port.